private/manageFishData.py

```diff
@@ -349,6 +349,19 @@ def is_big_fish(fish_id):
     return False
 
 
+def get_min_rating(fish_name):
+    try:
+        fsh_collectable_shop_refine = next((row['CollectablesShopRefine']
+                                            for row in XIV.game_data.get_sheet('CollectablesShopItem')
+                                            if str(row['Item']) == fish_name))
+    except StopIteration:
+        min_collect = 1
+    else:
+        key = int(str(fsh_collectable_shop_refine).split('#')[1])
+        min_collect = XIV.game_data.get_sheet('CollectablesShopRefine')[key]['LowCollectability']
+    return min_collect
```

Comment:
This looks a bit odd so I messed around in the API.  Instead of using the name, you can use the item key and reduce the code down to this.

A lot of the columns has `__str__` defined to return "Tablename#RowNumber", but the actual return value is the row itself.

```suggestion
def get_min_rating(item_id):
    shop_item = next(filter(lambda row: row.get_raw('Item') == item_id, XIV.game_data.get_sheet('CollectablesShopItem')))
    if shop_item is not None:
        return shop_item['CollectablesShopRefine']['LowCollectability']
    return 1
```

Reply:
That's much cleaner. I'll have to get the API downloaded so I can mess around with it to test things once the stuff going down with SaintCoinach merging to xivapi gets figured out.